src/ast/recfun_decl_plugin.h

```diff
@@ -302,7 +302,6 @@ namespace recfun {
         expr_ref_vector     m_args;
         case_expansion(recfun::util& u, app * n);
         case_expansion(case_expansion const & from);
```

Comment:
the copy constructor must go too. You can't have just one of them. Either all or nothing.